sirepo/package_data/static/json/jupyterhublogin-schema.json

```diff
@@ -5,5 +5,8 @@
                 "jupyterhublogin.js"
             ]
         }
+    },
+    "appDefaults": {
+        "route": "jupyterhublogin"
```

Comment:
Having trouble figuring out what the default route should be

Reply:
Does jupterhublogin work?

Reply:
It just hangs for me